vignettes/logistic-metadata-example.Rmd

```diff
@@ -317,11 +359,10 @@ me <- list(individualName = list(givenName = "Quinn",
 ### Coverage
 Set Taxonomic, Temporal, and Geographic Coverage.  (Look, apparently we're 
 modeling population densities of *Gloeotrichia echinulata* cyanobacterium in Lake Sunapee, NH, USA)
-
 ```{r}
 coverage <- 
-  set_coverage(begin = first(time), 
-               end = last(time),
+  set_coverage(begin = first(times), 
+               end = last(times),
                sci_names = "Gloeotrichia echinulata",
```

Comment:
Since this is a two species example can second species be added?  anabaena circinalis?